library/src/main/java/com/pengrad/telegrambot/model/giveaway/GiveawayCreated.kt

```diff
@@ -0,0 +1,5 @@
+package com.pengrad.telegrambot.model.giveaway
+
+data class GiveawayCreated(
+    @get:JvmName("prizeStarCount") val prizeStarCount: Int
```

Comment:
It says "Optional", perhaps should be `Int? = null` or `Int = 0`

Reply:
You're right, it should be Int = 0